darwin/exporter/formats/nifti.py

```diff
@@ -135,16 +143,19 @@ def check_for_error_and_return_imageid(video_annotation: dt.AnnotationFile, outp
 
     output_volumes = None
     filename = Path(video_annotation.filename)
-    suffixes = filename.suffixes
-    if len(suffixes) > 2:
-        return create_error_message_json(
-            "Misconfigured filename, contains too many suffixes", output_dir, str(filename)
-        )
-    elif len(suffixes) == 2:
+    try:
+        suffixes = filename.suffixes[-2:]
+    except IndexError:
+        suffixes = filename.suffixes
+    if len(suffixes) == 2:
         if suffixes[0] == ".nii" and suffixes[1] == ".gz":
-            image_id = str(filename).strip("".join(suffixes))
+            image_id = str(filename).rstrip("".join(suffixes))
```

Comment:
`strip()` was removing any character in any item in suffixes from the start and end of the string

Since we expect suffixes to contain ".nii", I noticed that for example a file beginning with "nyu" would become "yu"

`rstrip()` strips only trailing characters, which will be the suffixes